git/mockgit/mockgit.go

```diff
@@ -23,7 +23,7 @@ func (m *Mock) GitWithEditor(args string, output *string, editorCmd string) erro
 func (m *Mock) Git(args string, output *string) error {
 	fmt.Printf("CMD: git %s\n", args)
 
-	m.assert.NotEmpty(m.expectedCmd)
+	m.assert.NotEmpty(m.expectedCmd, fmt.Sprintf("Unexpected command: git %s\n", args))
```

Comment:
This provides a better error message that makes it easier to debug broken tests.